src/app/pages/home/components/task-calendar/task-calendar.component.html

```diff
@@ -47,7 +47,7 @@
           <div
             id="task-item"
             (click)="clicked(task)"
-            *ngIf="task.name !== 'ESM' && task.isClinical == false"
+            *ngIf="task.name !== 'ESM'"
```

Comment:
Maybe this can be added in the protocol too ? Will make it more flexible so we can add more tasks like ESM later on.

Reply:
I am also wondering why a scheduled clinical task is not shown in the calendar and also does not give a `Start` button ? Is it not exactly like a scheduled questionnaire ?

Reply:
> Maybe this can be added in the protocol too ? Will make it more flexible so we can add more tasks like ESM later on.
Yeah something like `showInTaskCalendar`? Or maybe this can be calculated if the `completionWindow` is less than an hour?

> I am also wondering why a scheduled clinical task is not shown in the calendar and also does not give a `Start` button ? Is it not exactly like a scheduled questionnaire ?
Yeah I am also not sure. I just assumed it was separate. Should this be added to the task calendar?

Reply:
> Yeah something like showInTaskCalendar? Or maybe this can be calculated if the completionWindow is less than an hour?

It would be preferable to have it in the protocol rather than the app as this can vary from project to project. Eg- some project may want to give user full insight into the scheduled and completed tasks while others (like currently in RADAR-CNS) may not expose the calendar for certain tasks.

> Yeah I am also not sure. I just assumed it was separate. Should this be added to the task calendar?

I am not sure about the rationale behind this either. But I think it should be added to the task calendar and also should display as a normal questionnaire with a Start button.

But i Think currently let's leave both of these as it is as this will require quite some refactoring. We can revisit this later. Maybe create an issue to track these and also the cache key issue?

Reply:
Okay added them here: #363